mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java

```diff
@@ -1,10 +1,14 @@
 package com.redhat.mercury.operator.utils;
 
+import com.redhat.mercury.operator.model.ApiVersion;
+
 import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.Map;
 
 public class ResourceUtils {
+    private static final Map<ApiVersion, ApiVersionPair> apiVersionMap = Map.of(ApiVersion.v1, new ApiVersionPair("v10", "v1"));
```

Comment:
why do you need to specify the openapi version when v1 routes are meant for mapping requests to v10?
I would remove this v10-v1 mapping and also the name from the openapi spec files